docs/README.md

```diff
@@ -24,7 +49,7 @@ The Contoso HR App is part of the Contoso platform and designed to serve interna
 
 ## App architecture
 
-To support this app functionality, the Contoso engineering team decided to build the app with the following:
+To support this app functionality, the Contoso engineering team decided to build the app with the following structure:
```

Comment:
I'll admit this is a "picky" point to make, but I'd word it with more confidence by removing "decided". Something like:

To support this app functionality, the Contoso engineering team used the following structure: